ckanext/unhcr/templates/user/dashboard_requests.html

```diff
@@ -0,0 +1,39 @@
+{% extends "user/dashboard.html" %}
+
+{% block dashboard_activity_stream_context %}
+{% endblock %}
+
+{% block primary_content_inner %}
+  <h2>{{ _('Data Containers') }}</h2>
+  {% if requests.containers %}
+    <hr>
+    <ul class="dataset-list unstyled">
+      {% for container in requests.containers %}
+        <li class="dataset-item">
+          <div class="dataset-content">
+            <h3 class="dataset-heading">
+              <a href="/data-container/{{ container.name }}">{{ container.display_name }}</a>
+            </h3>
+            {% if notes %}
```

Comment:
This is not showing the pending data container description 
```suggestion
            {% if container.description %}
```